Sources/SpeziContact/Models/ContactOption.swift

```diff
@@ -11,6 +11,43 @@ import SwiftUI
 
 
 /// Customizable way to get in contact with an individual and usually connected to a ``Contact``.
+///
+/// ### Usage
+///
+/// The following example demonstrates the usage of the ``ContactOption`` within a ``Contact``.
+/// ```swift
+/// Contact(
+///     name: PersonNameComponents(
+///         givenName: "First",
+///         familyName: "Last"
+///     ),
+///     image: Image(systemName: "figure.wave.circle"),
+///     title: "Title",
+///     description: "Description",
+///     organization: "Organization",
+///     address: {
+///         let address = CNMutablePostalAddress()
+///         address.country = "USA"
+///         address.state = "CA"
+///         address.postalCode = "94305"
+///         address.city = "City"
+///         address.street = "123 ABC St"
+///         return address
+///     }(),
+///     contactOptions: [
+///         .call("+1 (123) 456-7890"),
+///         .text("+1 (123) 456-7890"),
+///         .email(addresses: ["email@address.com"]),
+///          ContactOption(
```

Comment:
Indentation not correct here
Also, it would be a good idea to mention in a comment (like you did below) that this is the important part where a custom ContactOption is created